.circleci/config.yml

```diff
@@ -817,6 +822,15 @@ workflows:
           echidna_target: aliasing
           requires:
             - bedrock-echidna-build
+      - bedrock-echidna-run:
+          echidna_target: "burn:eth"
+          requires:
+            - bedrock-echidna-build
+      - bedrock-echidna-run:
+          echidna_target: "burn:gas"
+          size: 2xlarge
```

Comment:
the gas burn test seems to be a memory hog. This is the minimum size (32 gb) that will handle it.